tests/transforms/test_qcut.py

```diff
@@ -2095,7 +2112,7 @@ def circuit(x):
         res = circuit(x)
         assert np.allclose(res, np.cos(x))
         assert len(spy.call_args[0][0]) == 1  # there should be 2 tensors for wire 0
-        assert spy.call_args[0][0][0].shape == (4, 4)
+        assert spy.call_args[0][0][0].shape == ()
```

Comment:
Why did this have to change?

Reply:
The circuit in this test actually has a fully contained cut. Beforehand, we were going over all measurement and preparation configurations even though the cut didn't result in a disconnection. So we'd have a `(4, 4)` tensor with contraction equation `a,a`